backend-base/src/main/java/org/eclipse/che/incubator/workspace/telemetry/TelemetryResource.java

```diff
@@ -45,8 +45,10 @@ public String event(
       event.getProperties().stream()
         .collect(Collectors.toMap(e -> e.getId(), e -> e.getValue()));
 
+    AnalyticsEvent analyticsEvent = analyticsManager.transformEvent(AnalyticsEvent.valueOf(event.getId()), analyticsManager.getUserId());
+    analyticsManager.setCommonProperties(analyticsManager.transformProperties(params));
```

Comment:
I don't understand why properties coming from the current event are stored inside the `analyticsManager` in the `commonProperties` field.
Shouldn't the `commonProperties` field of the `analyticsManager` only contain constant properties (known from the beginning and common to all events) ? Then  I assume that the result of the merge of common properties + event-specific properties would be directly passed to the `onEvent` call.